frontends/web/src/components/password.tsx

```diff
@@ -211,18 +249,18 @@ class PasswordRepeatInputClass extends Component {
     }
   };
 
-  handleFormChange = event => {
-    let value = event.target.value;
+  handleFormChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    let value: string | boolean = event.target.value;
     if (event.target.type === 'checkbox') {
       value = event.target.checked;
     }
     const stateKey = event.target.id.slice(this.idPrefix().length);
-    this.setState({ [stateKey]: value }, this.validate);
+    this.setState({ [stateKey]: value } as Pick<TState, keyof TState>, this.validate);
   };
 
-  handleCheckCaps = event => {
+  handleCheckCaps = (event: KeyboardEvent) => {
     const capsLock = hasCaps(event);
-        if (capsLock != null) { // eslint-disable-line
+    if (capsLock != null) { // eslint-disable-line
```

Comment:
since you are here and this is now typed, it looks like ` // eslint-disable-line` isnt needed anymore
```suggestion
    if (capsLock !== null) {
```